proxy/iis.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: docs
+title: Setting up IIS
+permalink: /docs/proxy/iis/
+---
+
+> **NOTE**: This manual was made by using Windows Server 2012 with IIS 8. For other versions it might be slightly different.
+
+> **NOTE**: This manual was made by running Airsonic NOT under a context.
```

Comment:
What does that mean? Maybe add a link to the MS docs about what a context is?

Reply:
Ohh, with context I meant the Airsonic context not MS.

No context: http://airsonic.public.url 
Context: http://airsonic.public.url/**context**

Is there a way to describe this clearer?

Reply:
Ah, yes, that wasn't obvious, I thought it was some MS/IIS-specific thing. I generally refer to it as the "Airsonic context path" (potentially with a link to https://airsonic.github.io/docs/configure/standalone/ where this is documented).

Reply:
Updated in f87b609, and updated the website in airsonic/airsonic.github.io@15b05ea011db63d4f98f9931424cb3365c25af89. Should be live soon!